### DISCUSSION TITLE
VS Code help!!!

### DISCUSSION THREAD

**Discussion by GopiSVDev** [community/community#108916] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I don't know how do describe, but when i scroll it happens likes this(pic attached), i'm sure it was normal before, i don't know what i did.
i want it to be normal, please help. noob here

![Screenshot 2024-02-18 175909](https://github.com/community/community/assets/126976800/a1bdba32-8526-4a27-a41d-3934d60f1d25)
![Screenshot 2024-02-18 175440](https://github.com/community/community/assets/126976800/f275850b-c1b9-4720-971c-69f4235e75a4)

**Comment by albedim** [comment]
Hi, I don't think you did it by yourself, apparently it seems to be a VSC bug that i'm currently facing too.

**Reply by GopiSVDev** [reply]
What is it called? And If you get the solution please share me.

**Reply by albedim** [reply]
I don't know yet, i'll tell you something as soon as I know more.
Let me know if you find something

**Reply by GopiSVDev** [reply, marked as answer]
hey bro, i found the solution. just disable this
![Screenshot 2024-02-18 201830](https://github.com/community/community/assets/126976800/b1b706c7-e065-4c41-a1eb-b620690b3c56)

**Comment by CoreTiQ** [comment]
disable sticky scroll.

**Reply by GopiSVDev** [reply]
just did it. anyways thanks brother
https://github.com/orgs/community/discussions/108916#discussioncomment-8507683
